Diffing_Engine/Query/ListModifiedProperties.cs

```diff
@@ -40,6 +40,7 @@ public static partial class Query
         [MultiOutput(1, "propNames", "List of properties changed per each object.")]
         [MultiOutput(2, "value_current", "List of current values of the properties.")]
         [MultiOutput(3, "value_past", "List of past values of the properties.")]
+        [Deprecated("5.0", "Deprecated due to the new Diff object structure.")]
```

Comment:
This attribute has been deprecated - `ToBeRemoved` should be used instead

Reply:
Thanks, done 6a77ecb488d2af5ca5561d062af5c28b09c94293